src/components/OrderInput/index.tsx

```diff
@@ -82,7 +82,7 @@ export const OrderInput: React.FunctionComponent<OrderInputProps> = React.memo((
         <div className={cx}>
             <fieldset className={fieldsetFocusedClass}>
                 <CustomInput
-                    type="text"
+                    type="number"
```

Comment:
![image](https://user-images.githubusercontent.com/39953699/92604727-8310a880-f2b9-11ea-9ccd-bcf079443106.png)
this what happens when OrderInput has type number

Reply:
fixed, thanks👍